pkg/cmd/admin/migrate/migrator.go

```diff
@@ -212,7 +212,7 @@ func (o *ResourceOptions) Complete(f *clientcmd.Factory, c *cobra.Command) error
 		break
 	}
 
-	o.Builder = f.NewBuilder(false).
+	o.Builder = f.NewBuilder(true).
```

Comment:
Seems like this should be separate since it has nothing to do with dropping the `volumesource` command.